libs/nox/src/spatial.rs

```diff
@@ -1,3 +1,4 @@
+//! Provides abstractions for rigid body dynamics in 3D space.
```

Comment:
Can probably copy this paragraph from our Python API [docs](https://docs.elodin.systems/sdk/python-api#spatial-vector-algebra):

> Elodin uses Featherstone’s spatial vector algebra notation for rigid-body dynamics as it is a compact way of representing the state of a rigid body with six degrees of freedom. You can read a short into [here](https://homes.cs.washington.edu/~todorov/courses/amath533/FeatherstoneSlides.pdf) or in [Rigid Body Dynamics Algorithms (Featherstone - 2008)](https://link.springer.com/book/10.1007/978-1-4899-7560-7).